### PR TITLE
HeLx 1.1 deployment and bringing master up to date with develop 

### PR DESCRIPTION
**Pull request by ismagilani**

### CODE REVIEW

File: appstore/bin/appstore
Comment (pj-linebaugh) [inline appstore/bin/appstore]: Maybe just delete these?

**Review by pj-linebaugh — COMMENTED** [review]
Just noticed one place where code should probably just be deleted and not commented out.  Lots of delete files!

File: appstore/README.md
Comment (cschreep) [inline appstore/README.md:229]: Minor note, the capitalization of AppStore is inconsistent through this file (i.e. AppStore, appstore, Appstore)

File: appstore/bin/appstore
Comment (cschreep) [inline appstore/bin/appstore]: Any reason not to just remove the commented-out code?

**Comment by n0mn0m** [comment]
> Just noticed one place where code should probably just be deleted and not commented out. Lots of delete files!

For anybody curious about this during the dead code cleanup I also removed static files from being tracked in `git` since these can/should be created automatically. Today the flow that creates these is:

- Docker image is built with [bin/appstore start](https://github.com/helxplatform/appstore/blob/71d29dd5467c351b20e80f0bdf03f1e7df11f155/appstore/Dockerfile#L41) as its `CMD`.
- `bin/appstore start` runs [`updatedb`](https://github.com/helxplatform/appstore/blob/71d29dd5467c351b20e80f0bdf03f1e7df11f155/appstore/bin/appstore#L237) as part of it's function.
- In `updatedb` `collectstatic` is [called](https://github.com/helxplatform/appstore/blob/71d29dd5467c351b20e80f0bdf03f1e7df11f155/appstore/bin/appstore#L117) which will populate static from our apps as well as all `INSTALLED_APPS` which is helpful when we add new dependencies/apps with static assets.

Additionally jquery is coming from a CDN resources for all views in `core` now so we don't have those files tracked locally anymore.

File: appstore/bin/appstore
Comment (n0mn0m) [reply appstore/bin/appstore]: #73

File: appstore/README.md
Comment (n0mn0m) [reply appstore/README.md:229]: I can make an update that fixes the caps and captures the new front end setup being worked on in helxplatform/development#63 once Bo and I have the new image definition.

That would be after this merge of `develop` to `master`, but I think that would be ok?

Comment (cschreep) [reply appstore/README.md:229]: Sure, no need to hold up the release for some capitalization

File: appstore/bin/appstore
Comment (n0mn0m) [reply appstore/bin/appstore]: #73 merged

**Comment by n0mn0m** [comment]
@ismagilani this LGTM, for my understanding when this merges to `master` does that do a build with an image that goes to a stage environment before promotion into a production environment?

Also will/do we need to bump [`image_ver`](https://github.com/helxplatform/appstore/blob/e4e0b98ff901d3b9df8f37da064c6e08be1c6535/appstore/bin/appstore#L19)?

**Review by n0mn0m — APPROVED** [review]
🚀

**Review by cschreep — APPROVED** [review]

File: appstore/README.md
Comment (n0mn0m) [reply appstore/README.md:229]: helxplatform/development#643

**Comment by ismagilani** [comment]
> Just noticed one place where code should probably just be deleted and not commented out. Lots of delete files!

in bin/appstore?

**Comment by ismagilani** [comment]
> @ismagilani this LGTM, for my understanding when this merges to `master` does that do a build with an image that goes to a stage environment before promotion into a production environment?
> 
> Also will/do we need to bump [`image_ver`](https://github.com/helxplatform/appstore/blob/e4e0b98ff901d3b9df8f37da064c6e08be1c6535/appstore/bin/appstore#L19)?

What is LGTM? as for image_ver, it seems old this variable is used when image () is called to push different product images to github.  @pj-linebaugh do you know if we use this in helm charts? How do different appstore images get built and pushed to dockerhub?

**Comment by n0mn0m** [comment]
> > @ismagilani this LGTM, for my understanding when this merges to `master` does that do a build with an image that goes to a stage environment before promotion into a production environment?
> > Also will/do we need to bump [`image_ver`](https://github.com/helxplatform/appstore/blob/e4e0b98ff901d3b9df8f37da064c6e08be1c6535/appstore/bin/appstore#L19)?
> 
> What is LGTM? as for image_ver, it seems old this variable is used when image () is called to push different product images to github. @pj-linebaugh do you know if we use this in helm charts? How do different appstore images get built and pushed to dockerhb?

LGTM is short for Looks Good To Me. Thanks for the `image_ver` insight.

**Comment by ismagilani** [comment]
> > > @ismagilani this LGTM, for my understanding when this merges to `master` does that do a build with an image that goes to a stage environment before promotion into a production environment?
> > > Also will/do we need to bump [`image_ver`](https://github.com/helxplatform/appstore/blob/e4e0b98ff901d3b9df8f37da064c6e08be1c6535/appstore/bin/appstore#L19)?
> > 
> > 
> > What is LGTM? as for image_ver, it seems old this variable is used when image () is called to push different product images to github. @pj-linebaugh do you know if we use this in helm charts? How do different appstore images get built and pushed to dockerhub?
> 
> LGTM is short for Looks Good To Me. Thanks for the `image_ver` insight.

LOL, LOL, sorry about that @n0mn0m (smh smiley here)

**Pull request merged**
